node-api/handlers/beacon/types/conversions.go

```diff
@@ -73,3 +75,45 @@ func ValidatorFromConsensus(v *ctypes.Validator) *Validator {
 		WithdrawableEpoch:          v.GetWithdrawableEpoch().Base10(),
 	}
 }
+
+// useful in UTs
+func ValidatorToConsensus(v *Validator) (*ctypes.Validator, error) {
```

Comment:
TODO: add property tests to ensure `ValidatorToConsensus` and `ValidatorFromConsensus` are dual